hdrline.c

```diff
@@ -841,12 +841,15 @@ static const char *hdr_format_str(char *dest, size_t destlen, size_t col, int co
     case 'g':
       if (!optional)
       {
-        colorlen = add_index_color(dest, destlen, flags, MT_COLOR_INDEX_TAGS);
-        mutt_format_s(dest + colorlen, destlen - colorlen, prefix,
-                      nm_header_get_tags_transformed(hdr));
-        add_index_color(dest + colorlen, destlen - colorlen, flags, MT_COLOR_INDEX);
+        if (option(OPT_NOTMUCH_SHOW_TAGS))
```

Comment:
tags are no more a notmuch thing, so I suggest we rename OPT_NOTMUCH_SHOW_TAGS to OPT_SHOW_TAGS